.env.dev

```diff
@@ -1,9 +1,8 @@
 PORT=3000
 HTTPS=true
 
-REACT_APP_BLOCKNATIVE_DAPP_ID=

```

Comment:
What value will be for BLOCKNATIVE by default?

Reply:
Empty.